src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
```

Comment:
You can use 'for' loop here. Check [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class)

```suggestion
        for(i = 0, i < 3, i++) {
        System.out.println(new Lottery().getRandomBall().toString());
        }
```